app/views/pages/history.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="page-container">
+  <%= stylesheet_link_tag "home" %>
+  <h1>History</h1>
+  <br>
```

Comment:
A `<br>` is a line break character, and it shouldn't be used to increase spacing. CSS and margins would be better for this, if you think it is needed.

Example of a proper usage:
```html
<p>
  This is a haiku <br>
  Haikus are amazing, right? <br>
  Yes, they really are
</p>
```

Reply:
got it